js/site.js

```diff
@@ -0,0 +1,31 @@
+class Site{
+    constructor(domain, scoreFunction) {
+        this.domain = domain,
+        this.trackers = [],
```

Comment:
The temporary site score function uses this but for the most part it's not used.